_inc/client/pro-status/index.jsx

```diff
@@ -117,6 +117,18 @@ const ProStatus = React.createClass( {
 				);
 			}
 
+			if ( 'google-analytics' === feature && 'jetpack_business' !== sitePlan.product_slug ) {
+				return (
+					<Button
+						compact={ true }
+						primary={ true }
+						href={ 'https://wordpress.com/plans/' + this.props.siteRawUrl + '?feature=google-analytics' }
```

Comment:
@rodrigoi please check that the URLs we're using have changed, pointing to jetpack.com site where they're redirected https://github.com/Automattic/jetpack/pull/5859/files#diff-de917e31f2552d86c1d05bf45b190582

@richardmuscat should the `source` URL parameter for this be `upgrade-google-analytics`? It would result in a URL like
```
https://jetpack.com/redirect/?source=upgrade-google-analytics&site=' + this.props.siteRawUrl + '&feature=google-analytics
```
That said, I've tried that URL in the browser replacing my site and it's correctly resolve but once in Calypso, I realized that we don't have a Google Analytics feature to highlight in the column plans comparison.